CallAutomation_SimpleIvr/Program.cs

```diff
@@ -126,21 +113,30 @@ await client.GetCallConnection(@event.CallConnectionId)
             }
             else
             {
-                PlaySource invalidAudio = new FileSource(new Uri(audioBaseUrl + builder.Configuration["InvalidAudio"]));
+                PlaySource invalidAudio = new FileSource(new Uri(baseUri + builder.Configuration["InvalidAudio"]));
                 await client.GetCallConnection(@event.CallConnectionId).GetCallMedia().PlayToAllAsync(invalidAudio, audioPlayOptions);
             }
```

Comment:
There are too many if else if statments here, can we add some utility method and abstract this out? For example a method which will return the audio for a particular tone and the play that over the call ?

Reply:
for keeping it simple, everything put in single file.